app/javascript/components/agency-logo/component.spec.js

```diff
@@ -21,8 +20,10 @@ describe("<AgencyLogo />", () => {
         }
       }
     });
-    const { component } = setupMountedComponent(AgencyLogo, {}, state);
 
-    expect(component.find("div").at(2).prop("style")).to.have.property("backgroundImage", "url(logo-full.png)");
+    mountedComponent(<AgencyLogo />, state);
+    expect(screen.getByTestId('background')).toHaveStyle(`background-image: url(${"logo-icon.png"})`)
```

Comment:
change `background-image: url(${"logo-icon.png"})` to 'background-image: url("logo-full.png")'. No need to use template literals here.